src/components/Process/View.tsx

```diff
@@ -106,8 +106,19 @@ export const ProcessView = () => {
   }, [formErrors])
 
   return (
-    <Box>
-      <Box className='site-wrapper' mb={44}>
+    <Box
+      position='relative'
+      width='full'
+      m='0 auto'
+      maxW='1920px'
+      px={{
+        base: '10px',
+        sm: '20px',
+        md: '80px',
+      }}
+      py={{ base: '0px', sm: '15px', md: '30px', md2: '30px', lg: '30px' }}
```

Comment:
There's no need to repeat the same value after one definition. You should be able to safely remove md2 and lg from that list.